--- origsrc/hyperestraier-1.4.13/Makefile.in	2007-05-30 08:04:40.000000000 -0500
+++ src/hyperestraier-1.4.13/Makefile.in	2008-07-22 19:04:04.328125000 -0500
@@ -669,17 +669,14 @@
 	$(CC) -shared -Wl,-b -o $@ $(MYLIBOBJS)
 
 
-libestraier.dll.a : estraier.dll
+libestraier.dll.a : cygestraier-$(LIBVER).dll
 
 
-estraier.dll : $(MYLIBOBJS)
+cygestraier-$(LIBVER).dll : $(MYLIBOBJS)
 	$(CC) -shared -o $@ \
-	  -Wl,--out-implib=lib$@.a \
+	  -Wl,--out-implib=libestraier.dll.a \
 	  -Wl,--export-all-symbols \
 	  -Wl,--enable-auto-import \
-	  -Wl,--add-stdcall-alias \
-	  -Wl,--whole-archive \
-	  -Wl,--no-whole-archive \
 	  $(MYLIBOBJS) $(LDFLAGS) $(SKLIBS) $(MTLIBS) $(LIBS)
 
 
